gulpfile.js

```diff
@@ -34,13 +34,22 @@ const IMAGES_DIR = 'build/images';
 function parseCommandLine() {
   commander
     .option(
-      '--grep [pattern]',
+      '--grep [pattern...]',
       'Run only tests where filename matches a pattern'
```

Comment:
Related to this, one annoying issue at the moment is that this help documentation only exists in the code. If you run `gulp test --help` you get Gulp's help output, not the output for the specific command. It would be nice to fix that somehow.

Reply:
I have changed the help text as you suggested.

I will remove the options to allow multiple `--grep` options.

Regarding adding our own help documentation:
- it is been discussed in the gulp project: https://github.com/gulpjs/gulp/issues/2383
- another alternative: https://www.npmjs.com/package/gulp-display-help